tests/UI/pages/BO/orders/view/tabListBlock.js

```diff
@@ -432,12 +433,13 @@ class TabListBlock extends ViewOrderBasePage.constructor {
   /**
    * Set shipping details
    * @param page {Page} Browser tab
-   * @param shippingData {{carrier: string, shippingCost: string, trackingNumber: string}} Data to set on shipping form
+   * @param shippingData {{trackingNumber: string, carrier: string, carrierID: number}} Data to set on shipping form
    * @returns {Promise<string>}
    */
   async setShippingDetails(page, shippingData) {
     await this.setValue(page, this.trackingNumberInput, shippingData.trackingNumber);
-    await this.setValue(page, this.carrierSelect, shippingData.carrier);
+    await page.click(this.carrierSelect);
+    await this.waitForSelectorAndClick(page, `${this.carriersSelectResult} li:nth-child(${shippingData.carrierID})`);
```

Comment:
Could you.m1ke the menu item a selector please ?

Reply:
Done thx :+1: